tests/integration/test_proposals.py

```diff
@@ -0,0 +1,114 @@
+# Third Party Stuff
+import pytest
+from django.apps import apps
+from django.urls import reverse
+from tests import factories as f
+from rest_framework import status
+
+pytestmark = pytest.mark.django_db
+
+
+def test_proposal_creation_api(client):
+    Proposal = apps.get_model('proposals.Proposal')
+    url = reverse('proposal-list')
+    user = f.create_user()
+    data = {
+        'title': 'Test Proposal',
+        'kind': 'talk',
+        'level': 'beginner',
+        'duration': '00:30:00',
+        'abstract': 'Proposal abstract',
+        'description': 'Proposal description',
+    }
+    client.login(user)
+
+    response = client.post(url, data=data)
+    assert response.status_code == status.HTTP_201_CREATED
+    assert response.data["speaker"] == user.email
+    assert response.data["submitted_at"] is not None
```

Comment:
Why is `submitted_at` is `None`? When the proposal is created, it should contain the current time. No?

Reply:
This is okay. I missed the `not None` earlier.